src/YouTrackSharp/Infrastructure/Connection.cs

```diff
@@ -159,6 +159,11 @@ public void Post(string command, object data)
             MakePostRequest(command, data, HttpContentTypes.ApplicationXml);
         }
 
+        public void Post(string command, object data, string accept, string contentType)
+        {
+            MakePostRequest(command, data, accept, contentType);
+        }

```

Comment:
All of the payloads that I could find in this project were done through the URL exclusively without anything in the body. The time tracking API requires sending an XML payload with the request, so I abstracted the post routine to allow any content-type instead of the fixed `HttpContentTypes.ApplicationXWwwFormUrlEncoded` in `MakePostRequest()`. I had to modify `MakePostRequest()` somehow to allow to send XML data.

That's my reasoning but I'm not confident that the solution is the best approach.